examples/integrations/oci/vault-reactive/src/main/resources/application.yaml

```diff
@@ -14,9 +14,8 @@
 # limitations under the License.
 #
 
-# The values are read from
+# The values under oci.properties are read from
```

Comment:
Not sure what this means

Reply:
Just means that they can specify/override config values there and we read example.yaml as well in this app. Have clarified it.